stdlib/Random/docs/src/index.md

```diff
@@ -283,8 +283,10 @@ Note: `Sampler(rng, x)` is simply a shorthand for `Sampler(rng, x, Val(Inf))`, a
 The API is not clearly defined yet, but as a rule of thumb:
 1) any `rand` method producing "basic" types (`isbitstype` integer and floating types in `Base`)
    should be defined for this specific RNG, if they are needed;
-2) other documented `rand` methods accepting an `AbstractRNG` should work out of the box,
-   (provided the methods from 1) what are relied on are implemented),
+2) make sure that `copy` is possible for the new generator, otherwise, (i.e. hardware-based RNG ),
+   an error should be thrown;
+3) other documented `rand` methods accepting an `AbstractRNG` should work out of the box,
+   (provided the methods from 1) and 2) what are relied on are implemented),
```

Comment:
Here point 2) is not relevant for whether `rand` works. I suggest that you move your point 2 at the end of the list, i.e. as point 3, as for the other 2 existing points, it's the API of the `rand` function specifically which is discussed.